code/src/components/movieList.css

```diff
@@ -0,0 +1,168 @@
+.all-movies {
+  height: 100%;
+  width: 100%;
+  margin: 0;
+  background-repeat: no-repeat;
+  background-size: cover;
+  min-height: 100vh;
+  min-width: 300px;
+  line-height: 25px;
+  color: white;
+}
+
+.movie-info-box {
+  padding: 20px;
+  background-color: rgba(0, 0, 0, 0.281);
+  font-family: "Lucida Console", Courier, monospace;
+}
+
+.back-btn a {
+  font-size: 38px;
+  color: white;
+  text-decoration: none;
+  padding: 30px;
+  font-family: "Lucida Console", Courier, monospace;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.back-btn span {
+  text-decoration: none;
+  margin-left: 20px;
+  margin-bottom: 20px;
+}
+
+.path {
+  margin-top: 40px;
+}
+
+.back-btn svg {
+  margin-top: 40px;
+  transition: 0.4s ease-in-out;
+}
+
+.back-btn svg:hover {
+  background-color: greenyellow;
+  border-radius: 80%;
+  -webkit-animation: jello 0.9s infinite both;
+  animation: jello 0.9s infinite both;
+}
+
+@-webkit-keyframes jello {
+  0% {
+    -webkit-transform: scale3d(1, 1, 1);
+    transform: scale3d(1, 1, 1);
+  }
+  30% {
+    -webkit-transform: scale3d(1.25, 0.75, 1);
+    transform: scale3d(1.25, 0.75, 1);
+  }
+  40% {
+    -webkit-transform: scale3d(0.75, 1.25, 1);
+    transform: scale3d(0.75, 1.25, 1);
+  }
+  50% {
+    -webkit-transform: scale3d(1.15, 0.85, 1);
+    transform: scale3d(1.15, 0.85, 1);
+  }
+  65% {
+    -webkit-transform: scale3d(0.95, 1.05, 1);
+    transform: scale3d(0.95, 1.05, 1);
+  }
+  75% {
+    -webkit-transform: scale3d(1.05, 0.95, 1);
+    transform: scale3d(1.05, 0.95, 1);
+  }
+  100% {
+    -webkit-transform: scale3d(1, 1, 1);
+    transform: scale3d(1, 1, 1);
+  }
+}
+@keyframes jello {
+  0% {
+    -webkit-transform: scale3d(1, 1, 1);
+    transform: scale3d(1, 1, 1);
+  }
+  30% {
+    -webkit-transform: scale3d(1.25, 0.75, 1);
+    transform: scale3d(1.25, 0.75, 1);
+  }
+  40% {
+    -webkit-transform: scale3d(0.75, 1.25, 1);
+    transform: scale3d(0.75, 1.25, 1);
+  }
+  50% {
+    -webkit-transform: scale3d(1.15, 0.85, 1);
+    transform: scale3d(1.15, 0.85, 1);
+  }
+  65% {
+    -webkit-transform: scale3d(0.95, 1.05, 1);
+    transform: scale3d(0.95, 1.05, 1);
+  }
+  75% {
+    -webkit-transform: scale3d(1.05, 0.95, 1);
+    transform: scale3d(1.05, 0.95, 1);
+  }
+  100% {
+    -webkit-transform: scale3d(1, 1, 1);
+    transform: scale3d(1, 1, 1);
+  }
+}
```

Comment:
Whoa! I didn't notice this the first time I looked at the website, but nice Jello animation 💯